index.html

```diff
@@ -1,271 +1,325 @@
+              <li class="language-items">Javascript</li>
+              <li class="language-items">Ruby</li>
+              <li class="language-items">Html</li>
+              <li class="language-items">Css</li>
+            </ul>
+          </li>
+          <li class="about-skill-items">
+            <p class="skill-language">
+              <i
+                ><img src="assets/images/Rectangle3.svg" alt="" width="17"
+              /></i>
+              <strong class="language margin-05">Frameworks</strong>
+            </p>
+            <ul class="margin-top-10">
+              <li class="language-items">Bootstrap</li>
+              <li class="language-items">Ruby on Rails</li>
+              <li class="language-items">RSPec</li>
+              <li class="language-items">CapyBara</li>
+              <li class="language-items">Selenium</li>
+            </ul>
+          </li>
+          <li class="about-skill-items">
+            <p class="skill-language">
+              <i
+                ><img src="assets/images/Rectangle1.svg" alt="" width="17"
+              /></i>
+              <strong class="language margin-05">Skills</strong>
+            </p>
+            <ul class="margin-top-10">
+              <li class="language-items">Codekit</li>
+              <li class="language-items">Github</li>
+              <li class="language-items">Codepen</li>
+              <li class="language-items">Gitlab</li>
+              <li class="language-items">Terminal</li>
+            </ul>
+          </li>
+        </ul>
       </div>
-    </article>
-    <article class="card-work">
-      <div class="cards card-background-image">
-        <div class="card-name">
-          <h2 class="card-headline">Profesional Art Printing Data</h2>
+    </section>
+    <!-- contact-us -->
+    <footer id="contact-us">
+     <div class="contact-us">
+        <div class="contact-us-text">
+          <p class="contact-us-paragraph">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
-        <div class="card-text">
-          <p class="card-headline-text">A daily selection of privately personalized reads; no accounts or sign-ups
-            required. has been the industry's standard</p>
+       <form action="submit" method="post" id="form" class="form">
+        <div class="form-control">
+          <input type="text" placeholder="Full name" id="username" />
+          <div class="input-line"></div>
         </div>
-        <div class="card-skills">
-          <ul class="skills d-flex list-style-none">
-            <li class="skill2 skillcard">html</li>
-            <li class="skill3 skillcard">bootstrap</li>
-            <li class="skill4 skillcard">Ruby</li>
-          </ul>
-          <button class="card-button btn-cards">See Project</button>
+  
+        <div class="form-control">
+          <input type="text" placeholder="Email address" id="email" />
+          <div class="input-line"></div>
```

Comment:
- [ ] Kindly change the `type` attribute for the email form control to `email` and remove the `text` value. Doing this will make that particular form control accept inputs that are strictly in email format. 🙏